File: wlr/SPX.Core/GameManager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SPX.Core
{
    public static class GameManager
    {
        public static readonly int SpriteHeight = 32;
        public static readonly int SpriteWidth = 32;
        public static readonly int TileMapHeight = 20;
        public static readonly int TileMapWidth = 30;
        public static readonly float SpriteRadius = (float)Math.Sqrt(Math.Pow(SpriteHeight / 2, 2) + Math.Pow(SpriteWidth, 2));
    }
}

File: proj/src/spx/core/GameManager.java
package spx.core;

public class GameManager {
    public static float SpriteRadius = (float) Math.sqrt(Math.pow(Settings.get().spriteHeight / 2, 2) + Math.pow(Settings.get().spriteWidth, 2));
    public static int AnimationFps = 20;
}
